src/compass/utils/validate_product.py

```diff
@@ -10,7 +10,7 @@
 from osgeo import gdal
 
 
-DATA_ROOT = 'science/SENTINEL1'
+DATA_ROOT = 'CSLC'
 
 
```

Comment:
```suggestion
```
Replace with import from above

Reply:
Not sure if this is doable. The `DATA_PATH` does not point to `CSLC` but to the data group.

Reply:
I believe this cannot be replaced with 
`from compass.utils.h5_helpers import DATA_PATH` 

as the `DATA_PATH` import points to `CSLC/data` and not `CSLC`

Reply:
My bad, What do you think about adding "CSLC_PATH = 'CSLC' in `h5_helpers.py` to be portable elsewhere? Or renaming `DATA_ROOT` to `CSLC_ROOT` to avoid naming conflict since `/CSLC` as a path is only used [here](https://github.com/opera-adt/COMPASS/blob/09c56d53d586f9c8728655730d2fc38a96ee6843/src/compass/utils/validate_product.py#L301C41-L302)?